src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -73,6 +75,34 @@ def validate_create_parameters(namespace):
         raise CLIError('--dns-prefix has no value')
 
 
+def validate_ip_ranges(namespace):
+    if not namespace.api_server_authorized_ip_ranges:
+        return
+
+    restrict_traffic_to_agentnodes = "0.0.0.0/32"
+    allow_all_traffic = ""
+    ip_ranges = [ip.strip() for ip in namespace.api_server_authorized_ip_ranges.split(",")]
+
+    if restrict_traffic_to_agentnodes in ip_ranges and len(ip_ranges) > 1:
+        raise CLIError("--api-server-authorized-ip-ranges cannot restrict traffic via 0.0.0.0/32 and \
```

Comment:
Can we update the error message to be "Setting--api-server-authorized-ip-ranges to 0.0.0.0/32 is not allowed with other IP ranges. Refer to https://aka.ms/aks/whitelist for more details"? I think this is more direct and it matches with the error code in RP sync.

Reply:
Updated